packages/tocco-ui/src/SearchBox/SearchBox.spec.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import SearchBox from './SearchBox'
+import Button from '../Button'
+
+import {shallow} from 'enzyme'
+
+describe('tocco-ui', function() {
+  describe('SearchBox', function() {
+    const DEFAULT_DEBOUNCE = 200
```

Comment:
This is the default value of the SearchBox component. I would have to set it manually lower than 200 but then the default behaviour cannot be tested.